### PR TITLE
Improvements - pom.xml

### PR DESCRIPTION
**Pull request by darxriggs**
@lucamilanesio that's the pom.xml improvements part of pull request #27.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -87,30 +77,10 @@
...
-        <dependency>
-            <groupId>org.json</groupId>
-            <artifactId>json</artifactId>
-            <version>20140107</version>
-        </dependency>
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
-            <version>2.7.4</version>
+            <version>2.9.8</version>
```
Comment (maximecharron) [inline pom.xml]: Just to be sure, did you test this change? I wouldn't want it to break anything.

Comment (darxriggs) [reply pom.xml]: I expect [JSONUtilTest](https://github.com/jenkinsci/statistics-gatherer-plugin/blob/statistics-gatherer-1.1.3/src/test/java/org/jenkins/plugins/statistics/gatherer/util/JSONUtilTest.java) to accomplish this. I had a look and it covers a least all methods of `JSONUtil`.

Comment (darxriggs) [reply pom.xml]: The old 2.7.4 was released on 29-Apr-2016 (see https://github.com/FasterXML/jackson/wiki/Jackson-Releases). In the meantime a lot of security patches have been applied. Therefore I strongly suggest to upgrade this dependency.

**Comment by darxriggs** [comment]
@maximecharron / @lucamilanesio can it be merged?

**Comment by darxriggs** [comment]
@maximecharron / @lucamilanesio ping

**Comment by darxriggs** [comment]
Let me try another ping. Can this be merged @lucamilanesio?

**Comment by darxriggs** [comment]
@maximecharron / @lucamilanesio is this plugin not maintained anymore?

**Comment by lucamilanesio** [comment]
> @maximecharron / @lucamilanesio is this plugin not maintained anymore?

I would be more than happy to give you more permissions, I don't have a lot of free time to dedicate to this plugin anymore :-(

What do you think?

**Comment by darxriggs** [comment]
@lucamilanesio thanks for the offer, I really appreciate that. As I am trying to improve so many plugins, taking over maintenance of this one is not feasible. I am used to reviews so much that I anyway would need your support.

I'll try to split my original big pull request #27 into more pieces (there is already #29 and #35) and maybe sometime you will find the time to review them.

**Pull request merged**